linux/LinuxProcessList.c

```diff
@@ -1111,7 +1169,11 @@ static inline void LinuxProcessList_scanMemoryInfo(ProcessList* this) {
    char buffer[128];
    while (fgets(buffer, 128, file)) {
 
-      #define tryRead(label, variable) do { if (String_startsWith(buffer, label) && sscanf(buffer + strlen(label), " %32llu kB", variable)) { break; } } while(0)
+      #define tryRead(label, variable) \
```

Comment:
We should probably exit the case (*break*), if just the string was found (not necessarily data was parsed):

```c
#define tryRead(label, variable) \
         if (String_startsWith(buffer, label) { \
            sscanf(buffer + strlen(label), " %32llu kB", variable)) \
            break; \
         }

Also I like to align the trailing back-slashes

Reply:
Actually, the original macro has another bug: It should have been `!sscanf(...)` all along.

Reply:
Should be fixed in the new revision.